source/openapi-spec.yml

```
@@ -58,7 +65,43 @@ paths:
                     type: array
                     items:
                       $ref: "#/components/schemas/Application"
+        '401':
+          $ref: "#/components/responses/Unauthorized"
+
+        '404':
+          $ref: "#/components/responses/Notfound"
+
 components:
+  responses:
+    Notfound:
+      description: Not found
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              errors:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Error'
+                example:
+                  code: 404
+                  message: Unable to find Application(s)
+
+    Unauthorized:
+      description: Unauthorized
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              errors:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Error'
+                example:
+                  code: 401
+                  message: Please use correct auth token
```

```suggestion
                  message: Please provide a valid authentication token
```

Not really sure about my suggestion either tbh... 😅